src/devices/Ft4222/README.md

```diff
@@ -27,9 +27,25 @@ Alternatively, you can register your dll globally. Run from the directory where
 
 ## Linux Requirements
 
-For Linux, you need to download and install the proper version of ```libft4222.so```. The lib include the driver itself in a static binding. You can find the library [here](https://www.ftdichip.com/Support/SoftwareExamples/libft4222-linux-1.4.2.184.tgz). Once downloaded and extracted, run provided script ```install4222.sh``` as root. This will install the required library related to your system.
+For Linux, you need to download and install the proper version of ```libft4222.so```. The lib include the driver itself in a static binding. You can find the library [here](https://www.ftdichip.com/Support/SoftwareExamples/libft4222-linux-1.4.4.9.tgz). You can find the latest version [here](https://www.ftdichip.com/Products/ICs/FT4222H.html). Once downloaded and extracted, run provided script ```install4222.sh``` as root. This will install the required library related to your system.
 
-Alternately, you can copy your platform library into the same directory of your project if you don't want to install it. While writing this page, they all look like ```libft4222.so.1.4.2.184```. The library name needs ti be ```libft4222.so```.
+Alternately, you can copy your platform library into the same directory of your project if you don't want to install it. While writing this page, they all look like ```libft4222.so.1.4.2.184```. The library name needs to be ```libft4222.so```.
+
+## Mac OS Requirement
+
+For Mac OS, you'll need to download both the ```libft4222.dylib``` and the ```libftd2xx.dylib```. You can find the latest version of ```libft4222.dylib``` [here](https://www.ftdichip.com/Products/ICs/FT4222H.html), direct link on a known minimal working version [here](https://www.ftdichip.com/Support/SoftwareExamples/LibFT4222-mac-v1.4.4.14.zip). You can find the latest version of ```libftd2xx.dylib``` [here]https://www.ftdichip.com/Drivers/D2XX.htm.
+
+First install ```libftd2xx.dylib``` by follwing the instructions. To install ```libft4222.dylib``` you will need to follow the following steps. they are similar to the one installing ```libftd2xx.dylib```:
+
+From the path where you'll find the libft4222.x.x.x.x.dylib like for example: libfT4222.1.4.4.14.dylib
+```bash
+sudo cp libfT4222.1.4.4.14.dylib /usr/local/lib/libfT4222.1.4.4.14.dylib
+sudo ln -sf /usr/local/lib/libfT4222.1.4.4.14.dylib /usr/local/lib/libfT4222.dylib
+sudo cp /boost_libs/libboost_system.dylib /usr/local/lib/libboost_system.dylib
+sudo cp /boost_libs/libboost_thread-mt.dylib /usr/local/lib/libboost_thread-mt.dylib
+```
+
+This will install all what you need to then run properly any .NET Core IoT application using this specific FT4222 implementation. Be aware that at the first execution Mac OS will most likely reject the execution for security reasons. You'll have to manually go to the security settings and manually allow the libraries to execute. Once done, you will not be prompted again.
```

Comment:
Should we also add instructions on how to do this? Or is it pretty straight forward? I'm not a OSX use myself so I have to ask :P

Reply:
My experience is that the error message is very clear and tells you exactly what to do. You just have to click, it does open the security parameters at the right spot and you find a message saying you have to click to allow execution. It's annoying, as you have to click multiple times compare to the Windows experience where it does ask directly but it's ok and you can't really miss it. We have to keep in mind as well that most Mac OS users for this will be developers. So as this is happening quite often, they should know (or they'll have already disable this feature) how to do those steps.